examples/petsc/bps.c

```diff
@@ -77,16 +77,14 @@ static PetscInt GlobalStart(const PetscInt p[3], const PetscInt irank[3],
                             PetscInt degree, const PetscInt melem[3]) {
   PetscInt start = 0;
   // Dumb brute-force is easier to read
-  for (PetscInt i=0; i<p[0]; i++) {
-    for (PetscInt j=0; j<p[1]; j++) {
+  for (PetscInt i=0; i<p[0]; i++)
+    for (PetscInt j=0; j<p[1]; j++)
       for (PetscInt k=0; k<p[2]; k++) {
         PetscInt mnodes[3], ijkrank[] = {i,j,k};
         if (i == irank[0] && j == irank[1] && k == irank[2]) return start;
         GlobalNodes(p, ijkrank, degree, melem, mnodes);
         start += mnodes[0] * mnodes[1] * mnodes[2];
       }
-    }
-  }
```

Comment:
I guess I don't much care, but leaving the braces in is more conventional in this case, even though it eats up vertical space on your screen.

Reply:
I can revert it - we've just largely removed the braces when not strictly required.